scripts/development/k8s/elasticsearch-kibana/elasticsearch.yaml

```diff
@@ -16,10 +16,12 @@ spec:
     spec:
       containers:
         - name: elasticsearch
-          image: docker.elastic.co/elasticsearch/elasticsearch:7.6.0
+          image: docker.elastic.co/elasticsearch/elasticsearch:8.2.3
           env:
             - name: discovery.type
               value: single-node
+            - name: xpack.security.enabled
```

Comment:
I wonder if it's worth adding a comment above these ES settings highlighting that they are for demonstration purposes only.. Thoughts?